janitor/functions.py

```diff
@@ -3657,6 +3809,63 @@ def to_datetime(
     return df
 
 
+@pf.register_dataframe_method
+def trunc_datetime(datepart: str, timestamp: dt.datetime):
```

Comment:
Because this function is registered as a dataframe method through the `register_dataframe_method` decorator, it is missing the first argument `df`, and by extension, the column name on which this operation should be performed.

Reply:
On second look, I think this function is actually an "elementary function" (akin to the NumPy universal functions) that operate elementwise. 

Looking at the function body, I think the function as it stands is not yet a dataframe method. 

**Action items:** Please do the following:

1. Rename function to `truncate_datetime`.
2. Implement it such that it works with the pattern `df.truncate_datetime(...)`
3. Add a test for this one.